.gitignore

```diff
@@ -3,6 +3,8 @@
 *.iml
 .env
 .DS_Store
+taxi_service/settings.py
```

Comment:
why should you ignore settings.py?
It means if you for example update AUTH_USER_MODEL, then these changes won't be pushed to repo and thus won't be applied on your app